ES/ESdd041.xml

```diff
@@ -177,144 +184,131 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                              <locus target="#97"/>(white and orange).</term>
                         </decoNote>
-                        <decoNote xml:id="b3" type="Endbands"/>
-                        <decoNote xml:id="b4" type="Other">Rows of small holes are visible on the
-                           spine fold of the quires, one close to the head and one close to the tail
-                           of the codex.</decoNote>
-                        <decoNote xml:id="b5">None</decoNote>
                      </binding>
                   </bindingDesc>
                </physDesc>
+               
                <history>
                   <origin>
                      <origPlace>
                         <placeName ref="INS0105DD"/>
                      </origPlace>
-                     <origDate when="1601">According to the colophon, the Ms.
-                        was completed in 253 year of mercy (=1601 A.D.), during the reign of King
-                        Yāʿqob (r. 1597-1603, 1605-07) and the tenure of Metropolitan Ṗeṭros
-                        (1599-1606). Patriarch of Alexandria ( liqa ʾepisqoṗosat) Gabrǝʾel (Gabriel
-                        VIII, in tenure 1587-1603) and Metropolitan Ṗeṭros (ṗāṗās) are mentioned on
-                           <locus target="#33va">33va</locus> and 33vb, respectively.</origDate>
+                     <origDate when="1601">According to the colophon, the manuscript  was completed in <date calendar="diocletian" when="1601" when-custom="253" evidence="internal">253 year of mercy, i.e. 1601 CE</date>, 
+                        during the reign of <persName ref="PRS10198Yaeqob"><roleName type="title">King</roleName> Yāʿqob</persName> (r. <date>1597-1603, 1605-07</date>) and the tenure of 
+                        <persName ref="PRS7829Petros"><roleName type="office">Metropolitan</roleName> Ṗeṭros</persName> (<date>1599?–1606</date>). 
+                        Patriarch of Alexandria ( liqa ʾepisqoṗosat) <persName ref="PRS4381Gabriel"> Gabrǝʾel (Gabriel VIII)</persName>,
+                        in tenure <date>1587–1603</date>) and <persName ref="PRS7829Petros"><roleName type="office">Metropolitan (ṗāṗās)</roleName> Ṗeṭros</persName> (<date>1599?–1606</date>) 
+                        are mentioned on <locus target="#33va"/> and <locus target="#33vb"/>, respectively.</origDate>
                   </origin>
-                  <provenance>The commissioner is ʾabbā Za-Krǝstos mentioned in the supplication formula on <locus target="#3ra">3ra</locus> and in the colophon.</provenance>
+                  <provenance>The commissioner is <persName ref="PRS14617Zakrestos_" ><roleName type="title">ʾabbā</roleName> Zakrǝstos</persName> mentioned in the supplication formula on <locus target="#3ra"/> and in the <ref target="#coloph1"/>.</provenance>
```

Comment:
Ask Zhenia about PRS14617Zakrestos_, the dash should not be at the end.

Reply:
I have created an issue

Reply:
always link issues https://github.com/BetaMasaheft/Documentation/issues/2741